engine/src/object.h

```diff
@@ -1268,6 +1315,17 @@ class MCObject : public MCDLlist
 	bool getdeletionissuspended(void) const { return m_deletion_is_suspended; }
 	void setdeletionissuspended(bool value) { m_deletion_is_suspended = value; }
 
+    // Because C++11 support is universal amongst our supported platforms, auto-
```

Comment:
@livecodefraser should there be a `not` before universal... otherwise I don't get this comment... but that could just be me ;-)